server/Dockerfile

```diff
@@ -1,3 +1,12 @@
+FROM gradle:jdk11 as builder
+
+COPY --chown=gradle:gradle . /home/gradle
```

Comment:
Can anything bad happen by copying directly into the home directory? Shall we use a subdir like `/home/gradle/openvsx`?

Reply:
The `home/gradle` directory only contains a `.gradle` folder. I am not aware of any drawbacks which could be caused by that folder. However, if you prefer a subdirectory anyway, I'll change the folder structure.

Reply:
Ok, let's keep it until we see an actual problem.